content/english/demo.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Demo instance
+description: Information about the demo instance
+draft: false
+---
+
+The demo instance is a publicly available instance for trying out HedgeDoc and can be found at [demo.hedgedoc.org](https://demo.hedgedoc.org).
+
+While notes and accounts on the demo instance don't get deleted periodically, **you should not rely on the demo instance for production use**.  
+**We cannot guarantee that your notes won't be lost** because of hardware or software malfunctions.
+This is mainly due to the fact that this instance runs the latest unstable code from our repository.
+
+Some countries have a too high latency to the demo instance, which means HedgeDoc would be unusable. To make the demo available in as many countries as possible, we use [Cloudflare](https://www.cloudflare.com/). One of the [downsides](https://en.wikipedia.org/wiki/Cloudflare#Controversy) is that Cloudflare intercepts all TLS traffic between your browser and the demo instance, which makes the demo unsuitable for private notes.
```

Comment:
I think this sentence needs some rearrangement. Maybe start with `To make the demo available…`